File: online-judge-solutions/Codeforces/1708A.py
#https://codeforces.com/problemset/problem/1708/A
#Lang		:	Python 3.8
#Time		:	46 ms
#Memory         :	0 KB
for _ in range(int(input())):
    n,b=int(input()),True
    v=list(map(int,input().split()))[:n]
    b=[False for i in range(1,len(v),1) if v[i]%v[0]!=0]
    print("YES" if not b else "NO")

File: online-judge-solutions/Codeforces/1708A.cpp
//https://codeforces.com/problemset/problem/1708/A
//Lang		:	GNU C++14
//Time		:	31 ms
//Memory	:	0 KB
#include<iostream>
using namespace std;
int main(){
	int t,n;
	cin>>t;
	while(t--){
		cin>>n;
		bool b=true;
		int v[n];
		for(int i=0;i<n;i++){
			cin>>v[i];
			if(i>0 && v[i]%v[0]!=0){b=false;}
		}
		cout<<(b?"YES":"NO")<<"\n";
	}
	return 0;
}
